modules/kubernetes-addons/karpenter/locals.tf

```diff
@@ -17,7 +17,7 @@ locals {
     name        = local.name
     chart       = local.name
     repository  = "https://charts.karpenter.sh"
-    version     = "0.14.0"
+    version     = "0.15.0"
```

Comment:
I believe there is 0.16.x already

Reply:
I have not tried it with eks-blueprints. 
I tried it with an eks 1.23 cluster via helm but it had some issues I have not reported.
0.15.0 works.

Reply:
been using it with EKS 1.23 and 1.22 with karpenter 0.16 without any issues since it was released.